changes/2130.misc.rst

```diff
@@ -0,0 +1 @@
+The set_font() method has been implemented for Android
```

Comment:
It's specific to selection, not "whole of android".

Reply:
OK, changed the text accordingly